routes/api.php

```diff
@@ -73,6 +73,7 @@
         Route::put('rules', 'LegacyApiController@add_edit_rule')->name('edit_rule');
         Route::delete('rules/{id}', 'LegacyApiController@delete_rule')->name('delete_rule');
         Route::post('services/{hostname}', 'LegacyApiController@add_service_for_host')->name('add_service_for_host');
+        Route::get('oxidized/configsearch/{searchstring}', 'LegacyApiController@search_oxidized')->name('search_oxidized');
```

Comment:
Any reason why you added this under the admin middleware?  
As far as I know you don't need admin in the web interface for oxidized

Reply:
Thanks for replying :)

The reason behind it was that getting to search through config files is something i would imagine only admins should have access to.

I can move it anywhere you want it to be :)

Reply:
You do need to be an admin.  Many want it able to be separated, but that isn't an option yet.